packages/hardhat-core/test/internal/hardhat-network/stack-traces/compilers-list.ts

```diff
@@ -295,6 +295,20 @@ export const solidityCompilers: SolidityCompiler[] = [
     },
     latestSolcVersion: true,
   },
+  {
+    solidityVersion: "0.8.28",
+    compilerPath: "soljson-v0.8.28+commit.7893614a.js",
+    latestSolcVersion: true,
```

Comment:
Do yo think we should we remove that setting (`latestSolcVersion = true`) from versions `0.8.25` through `0.8.27`? This settings controls which versions are tested in the CI by default. I think we should at least remove it from `0.8.25` so that we don't start growing the number of compilers being tested.